src/frontend/src/components/public/PIAFormTabs/collectionUseAndDisclosure/index.tsx

```diff
@@ -54,21 +55,25 @@ const PIACollectionUseAndDisclosure = () => {
     {
       key: 'drafterInput',
       label: Messages.WorkThroughDetails.ColumnDrafterInput.en,
+      type: TextInputType.INPUTTEXTAREA,
     },
     {
       key: 'mpoInput',
       label: Messages.WorkThroughDetails.ColumnMpoInput.en,
       isDisable: !isMPORole(),
+      type: TextInputType.INPUTTEXT,
```

Comment:
(optional): can we have type default to INPUT_TEXT?

Reply:
I think it is best to be explicit as most likely users are going to demand many of the inputs to be multi-line. The reason why we have this ticket is that they struggle to type everything in one line. I would expect them to ask for more inputs that are single line to multi-line. We may have input used sparingly and textarea used more. 

Better to be explicit I think. Thoughts?

Reply:
for readability, INPUT_TEXT looks better. Agree.

Reply:
I am fine either ways, and hence put (optional). The reason I suggest is because most of our fields at the moment are TEXT and not TEXT_AREA, but if you anticipate that change over time, happy with being explicit.